src/test/java/com/jakewharton/rxrelay/PublishRelayTest.java

```diff
@@ -42,15 +42,13 @@
 
     relay.call("one");
     relay.call("two");
-    relay.call("three");
 
     @SuppressWarnings("unchecked") Observer<String> anotherObserver = mock(Observer.class);
     relay.subscribe(anotherObserver);
 
-    relay.call("four");
+    relay.call("three");
```

Comment:
Is there any reason you changed this? It's still 1-3 and then 4 in RxJava which is where all these tests are from: https://github.com/ReactiveX/RxJava/blob/1f65cbbceea6d152b8a0f5914eea82eba4686c48/src/test/java/rx/subjects/PublishSubjectTest.java#L60